requirements-dev.txt

```diff
@@ -0,0 +1,3 @@
+pip-tools==7.4.1
```

Comment:
Moved dev dependencies and ruff.toml to the repository root (they're common for all sub-projects)